test/postControllerSuperTest.js

```diff
@@ -0,0 +1,42 @@
+/* global it, describe */
+const request = require('supertest');
+
+const app = require('../app');
```

Comment:
You need to mock out the dependencies of postController otherwise you will require database, etc